src/components/ProgressIndicator/ProgressIndicatorDisplay.js

```diff
@@ -173,6 +175,18 @@ const ProgressIndicatorDisplay = ({
               isCurrent={step.isCurrent}
             />
           ))}
+          {showAppointment && (
+            <ProgressItem completed={isAppointment}>
+              <LinkOrSpan
+                to={'/appointment'}
+                useLink={areApplicableStepsCompleted}
```

Comment:
since there are no applicableSteps at all, I think this should be either always `true` or depend on the state of the "inloggen"/"start" step, however I'm not sure we can tackle this right now due to the rest of the flow not being baked in yet and functional decisions needing to be made about the inloggen/start step.